specutils/tests/test_spectrum1d.py

```diff
@@ -107,6 +108,20 @@ def test_create_with_spectral_coord():
     assert spec.spectral_axis.size == 50
 
 
+def test_create_from_cube():
+
+    flux = np.arange(24).reshape([2,3,4])*u.Jy
+    wcs_dict = {"CTYPE1": "RA---TAN", "CTYPE2": "DEC--TAN", "CTYPE3": "WAVE-LOG",
+                "CRVAL1": 205, "CRVAL2": 27, "CRVAL3": 3.622e-7,
+                "CRDELT1": -0.0001, "CRDELT2": 0.0001, "CRDELT3": 8e-11,
```

Comment:
typo (necessary for the suggestion below to work)
```suggestion
                "CDELT1": -0.0001, "CDELT2": 0.0001, "CDELT3": 8e-11,
```